content/blogposts/2022/release-4.2.md

```diff
@@ -0,0 +1,70 @@
+    url: https://github.com/sourcegraph/sourcegraph/pull/44467
+    category: Admin
+
+  - description: The search input has a new search history button, and search history can be cycled via the up/down arrow keys for quick access to previous searches.
+    url: https://github.com/sourcegraph/sourcegraph/pull/44544
+    category: Search
+  - description: "When the `content:` filter is used in a query, it now behaves more predictably and only saerches file contents. Previously, file contents, paths, and repos were searched. However, as before, if `type:` is also set, the `content:` filter will search for results of the specified `type:`."
+    url: https://github.com/sourcegraph/sourcegraph/pull/43442
+    category: Search
+  - description: When rendering a file which is backed by Git LFS, Sourcegraph now displays a page that links directly to the file on the codehost. Previously we rendered the LFS pointer.
+    url: https://github.com/sourcegraph/sourcegraph/pull/43686
+    category: Search
+  - description: Code Insights data points that do not contain any results will display zero instead of being omitted from the visualization for clarity. This only applies to insight data created after 4.2.
+    url: https://github.com/sourcegraph/sourcegraph/pull/43166
+    category: Code Insights
+  - description: "A new experimental GraphQL query, `permissionsSyncJobs`, now lists the states of recently completed permissions sync jobs and the state of each provider to check that syncing is working as intended. The TTL of entries can be configured with `authz.syncJobsRecordsTTL`."
+    url: https://github.com/sourcegraph/sourcegraph/pull/44387
+    category: API
+  - description: Repositories can now be ordered by size on the repo admin page to make it easy to find the largest synced repos.
+    url: https://github.com/sourcegraph/sourcegraph/pull/44360
+    category: Admin
+---
+
+Sourcegraph 4.2 is here! This month, we are kicking off significant work to improve the scale, speed, and security of Sourcegraph.
+
+The Sourcegraph platform is built to make developers' lives easier. With a great code intelligence platform, developers are able to find and understand code fast, stay in flow, and focus more on what they love doing: writing great code. In order to do this, Sourcegraph needs to fit the requirements of all sizes of engineering teams. These requirements include operating across massive numbers of repositories, being able to search millions of commits, and baking security into every part of the product.
+
+To accomplish these things and deliver a product for even the most demanding engineering teams, Sourcegraph is putting extra focus on scale, speed, and security, and you will see these themes surface frequently over the next several releases.
+
+<Badge link="https://docs.sourcegraph.com/admin/audit_log#on-premises" text="Admin" color="violet" size="small" />
+
+#### Get enhanced security visibility with audit logs
```

Comment:
@vrto Could you take a look at this highlight please?

Reply:
LGTM 👍